features/admin.template-core.v1/components/forms/handlers/validation/use-validation-handlers.tsx

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
```

Comment:
Move this code into hooks and utils

Reply:
https://github.com/wso2/identity-apps/pull/5896/commits/71394fd2f5a42fe3ee5be8609710c101fba5cea7 and https://github.com/wso2/identity-apps/pull/5896/commits/5ad79b46517a798145049d6e85cb5eafa40d3065